pycognito/__init__.py

```diff
@@ -223,7 +223,11 @@ def verify_token(self, token, id_name, token_use):
         unverified_claims = jwt.get_unverified_claims(token)
         token_use_verified = unverified_claims.get("token_use") == token_use
         if not token_use_verified:
-            raise TokenVerificationException("Your {} token use could not be verified.")
+            raise TokenVerificationException(
+                "Your {!r} token use ({!r}) could not be verified.".format(
+                    id_name, token_use
+                )
+            )
```

Comment:
Let's use the new faster f"" strings. `f"Your {id_name!r} ...."`

Reply:
Great! Done. The only reason I didn't use f-strings was because I didn't see them used anywhere else in the codebase and wanted to match the existing style.